### PR TITLE
KAZOO-244: more service plan processing

### PR DESCRIPTION
**Pull request by jamesaimonetti**
@k-anderson unless you find issues, of course.

### CODE REVIEW

**Comment by k-anderson** [comment]
@jamesaimonetti hmm found some issues when adding devices to a sub-account of a reseller.  It did not add the audit to the sub-account MODb and the audit it added to the reseller MODb has not quantities for the sub-account (also lists the diff as 2 devices which is odd).  Give me a call if you want to discuss!

File: applications/crossbar/src/crossbar_util.erl [hunk 1]
```diff
@@ -997,7 +997,6 @@ get_account_token_restrictions(AccountId, Method) ->
 
 -spec get_priv_level_restrictions(api_object(), api_binary()) -> api_object().
 get_priv_level_restrictions('undefined', _PrivLevel) -> 'undefined';
-get_priv_level_restrictions(_Restrictions, 'undefined') -> 'undefined';
```
Comment (fenollp) [inline applications/crossbar/src/crossbar_util.erl]: Update the spec too?

File: core/whistle_services-1.0.0/doc/lifecycle.md [hunk 2]
```diff
@@ -122,3 +122,131 @@ Two APIs exist to force reconciliation/syncing:
...
+         }
+     }
+     ,"error":"402"
+     ,"message":"accept charges"
+     ,"status":"error"
+     ,"request_id":"{REQUEST_ID}"
+     ,"auth_token":"{AUTH_TOKEN}"
+    }
+
+The payload shows the details of what the change would entail (in absolute terms). If the request is resubmitted with the `accept_charges` flag, and the account is in good standing, the device is created and the services for **R1** is updated.
```
Comment (fenollp) [inline core/whistle_services-1.0.0/doc/lifecycle.md]: s/is updated/are updated/ ?

**Comment by jamesaimonetti** [comment]
@k-anderson rebased

**Comment by fenollp** [comment]
@jamesaimonetti Needs rebase again

**Comment by jamesaimonetti** [comment]
@lazedo @k-anderson if travis agrees, this is rebased and ready to go.

**Pull request merged**